### PR TITLE
Added storing without score storing, changed styling

### PR DESCRIPTION
**Pull request by SvitlanaMOliinyk**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 import { quizData } from './data.js';
 import { initWelcomePage } from './pages/welcomePage.js';
+import { initQuestionPage } from './pages/questionPage.js';
 
 const loadApp = () => {
+  let storedQuestionNumber = JSON.parse(localStorage.getItem('questionNumber'));
```
Comment (arkaitzgarro) [inline src/app.js:6]: This can be `const`, because the value does not change.

**Review by arkaitzgarro — COMMENTED** [review]
Please run `npm run prettier` and we are good to go.

**Comment by arkaitzgarro** [comment]
@SvitlanaMOliinyk do you need help fixing the merge errors before merging?

**Comment by SvitlanaMOliinyk** [comment]
Yes, thank you

File: src/app.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
-  initWelcomePage();
+  const storedQuestionNumber = JSON.parse(
+    localStorage.getItem('questionNumber')
+  );
+  
+  if (
+    storedQuestionNumber === null ||
+    storedQuestionNumber === quizData.questions.length
+  ) {
+    quizData.currentQuestionIndex = 0;
```
Comment (arkaitzgarro) [inline src/app.js:14]: Let's also clear the localStorage to start the quiz fresh.

```
localStorage.clear();
```

**Comment by arkaitzgarro** [comment]
Implemented in #43

**Pull request closed without merging**